core/backend/src/BriefcaseManager.ts

```diff
@@ -76,6 +76,18 @@ export type PullChangesArgs = ToChangesetArgs & {
   onProgress?: ProgressFunction;
 };
 
+/** Arguments for [[BriefcaseManager.revertTimelineChanges]]
+ * @public
+ */
+export type RevertChangesArgs = Optional<PushChangesArgs, "description"> & {
+  /** If present, a function called periodically during the download to indicate progress.
+   * @note return non-zero from this function to abort the download.
+   */
+  onProgress?: ProgressFunction;
+  toIndex: ChangesetIndex;
```

Comment:
Please document all properties of this type.

Reply:
done